BRTesterMac/mac/Sources/main.m

```diff
@@ -0,0 +1,17 @@
+//
+//  main.m
+//  Brisk
+//
+//  Created by Rauan Mayemir on 6/29/18.
+//  Copyright © 2018 Brisk Contributors. All rights reserved.
+//
+
+#import <caml/callback.h>
+
+int main(int argc, const char *argv[])
+{
+  caml_main((char_os **) argv);
+  value *react_run = caml_named_value("React.run");
```

Comment:
Why do we have explicit run?

Reply:
I preferred `caml_main` with callback to our improvised entry point to `caml_startup` with top-level unit call.

Reply:
but truth be told we still have the top level unit call, but now there's another layer of indirection - the global thing registers a callback which will be in turn called by something else ?

Reply:
> a callback which will be in turn called by something else

The point here was to show that it's possible to plug brisk into an existing app. It doesn't say much at the moment, but I'm open to ideas.

(We could bring back AppDelegate and generic ViewController, that way you won't have to deal with NSApplication, but I want to hold off on that)

Reply:
yes, I'm for removing this stuff and moving it to the C side. It's actually closer to existing app environment 😄

Reply:
I'll leave the `React.run` named callback and add a discussion issue to move the AppDelegate to C side. (though I'm already convinced that it's the most likely real-world scenario)

Reply:
Filed #21.